Wallet Download and Installation.md

```diff
@@ -0,0 +1,77 @@
+# Downloading and Installing Wallets to Your Devices
+
+Visit the **[Downloads](https://www.skycoin.net/downloads/)** page on skycoin.net to download the Skycoin Desktop Wallet and Skycoin Mobile Wallet for the device you are using:
+
+## Desktop_Wallet
+
+* ### Windows
+* ### macOS
+* ### Linux
+
+## Mobile_Wallet
+
+* ### iOS
+* ### Android
+
+## Desktop Wallet
+
+### Windows
+
+You can follow these steps to install the Skycoin Desktop Wallet on your PC:
+
+* Click on the download link next to "Windows Installer (Recommended)" to start downloading the Wallet installer.
+
+* Once the installer has been downloaded, double click to open it.
```

Comment:
The comma here may be right but if it is, then there should be one on line 28 as well. Please check.